setup.py

```diff
@@ -16,17 +16,28 @@
         'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python :: 3.6',
     ],
-    packages=['.', 'config', 'endpoints', 'schemas', 'utils'],
+    packages=['aggregator', 'aggregator/config', 'aggregator/endpoints',
+              'aggregator/utils', 'registry', 'registry/config',
+              'registry/endpoints', 'registry/schemas',
+              'registry/utils'],
     package_data={'': ['*.json', '*.ini']},
     install_requires=[
         'aiohttp', 'asyncpg', 'aiohttp_cors', 'uvloop',
         'asyncio', 'aiocache', 'aiomcache', 'ujson',
-        'jsonschema==3.0.0a3'
+        'jsonschema==3.0.1', 'gunicorn'

```

Comment:
in
```suggestion
        'jsonschema==3.0.2', 'gunicorn'
```

to be on the same version as `requirements.txt`